app/lib/katello/capsule_content.rb

```diff
@@ -13,7 +13,26 @@ def lifecycle_environments(organization_id = nil)
       scope
     end
 
-    def pulp_repos(environments = lifecycle_environments, content_view = nil)
+    def pulp_server
+      uri = URI.parse(self.capsule.url)
+
+      Runcible::Instance.new(
+        :url      => "#{uri.scheme}://#{uri.host.downcase}",
+        :api_path => "/pulp/api/v2/",
+        :user     => "admin",
+        :timeout      => SETTINGS[:katello][:rest_client_timeout],
+        :open_timeout => SETTINGS[:katello][:rest_client_timeout],
+        :cert_auth => {:ssl_client_cert => ::Katello::Cert::Ca.ssl_client_cert,
+                       :ssl_client_key => ::Katello::Cert::Ca.ssl_client_key},
+        :logging  => {:logger     => ::Foreman::Logging.logger('katello/pulp_rest'),
+                      :exception  => true,
+                      :info       => true,
+                      :debug      => true }
+      )
+    end
```

Comment:
Perhaps you bring in this idea (https://github.com/ehelms/katello/blob/fixes-13470/app/services/katello/pulp/server.rb) to this PR, with configuration, and then I update my PR based on what comes out of this? The alternative is I update mine to your needs and then you update once mine goes in but I am OK with this functionality getting in first.